shared-bindings/bitbangio/SPI.c

```diff
@@ -43,19 +43,19 @@
 //|     """A 3-4 wire serial protocol
 //|
 //|     SPI is a serial protocol that has exclusive pins for data in and out of the
-//|     master.  It is typically faster than :py:class:`~bitbangio.I2C` because a
-//|     separate pin is used to control the active slave rather than a transmitted
+//|     microcontroller.  It is typically faster than :py:class:`~bitbangio.I2C` because a
```

Comment:
Have you seen discussion that replaces master with microcontroller? I think `main` might be clearer because `microcontroller` applies less well to a raspberry pi.